### ISSUE TITLE
WC_Shipping_Zones::get_zones() not working

### ISSUE THREAD

**Issue by jkourou** [woocommerce/woocommerce#21279]
Hello.

This bug I have reproduced in
1) WP 4.8.7 and WC 3.1.2 
2) WP 4.9.8 and WC 3.4.5

In the (2) scenario the only plugin is WC and we are on default Storefront theme.

When I call `WC_Shipping_Zones::get_zones()`, I get "**Call to a member function get_continents() on null at line 111 of class-wc-shipping-zone.php**"

In line 111 at [class-wc-shipping-zone.php](https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-shipping-zone.php) we see that there is a call at `WC()->countries->get_continents()`. Calling `WC()->countries` by itself returns NULL.

I could not find any similar issue so I resulted to post it here.

The rest of the `WC_Shipping_Zones` methods work fine.

Thank you.

**Comment by kloon** [comment]
Where are you running this code? When I test it using WP Shell it returns results fine. I suspect it might be because you are calling it before WooCommerce is actually loaded.

**Comment by jkourou** [comment]
In an included PHP file, called from the theme's functions.php

**Comment by kloon** [comment]
OK, that may be why. You will need to tie it to a hook instead, difficult to give an example as I do not know what you are trying to achieve. Try using the woocommerce_loaded hook, that should definitely work.

**Comment by jkourou** [comment]
@kloon 
Unfortunately the `woocommerce_loaded` and `plugins_loaded` hooks you cannot add actions from the theme's **functions.php**, because they are not executed (too late).

Also I tried using Storefront's `storefront_is_woocommerce_activated`, but this led to the bug I originally mentioned.

I ended up using `wp_loaded` which works from the theme's functions.php and also the `WC_Shipping_Zones::get_zones()` executed successfully.
